op-monitorism/multisig/monitor.go

```diff
@@ -170,6 +180,7 @@ func (m *Monitor) checkPresignedNonce(ctx context.Context) {
 			nonce, err := strconv.ParseInt(item.Title[6:len(item.Title)-5], 10, 64)
 			if err != nil {
 				m.log.Error("failed to parse nonce from item title", "title", item.Title)
+				m.unexpectedRpcErrors.WithLabelValues("1pass", "title").Inc()
```

Comment:
NITS: Usage of magic numbers for (`6`, `5`, `10`) should we remove them to improve readibility? 
```go
180                 nonce, err := strconv.ParseInt(item.Title[6:len(item.Title)-5], 10, 64) // Should we remove the magic number maybe overkill but improve readability in the long run. 
```

Reply:
yea sounds good i'll create some variables